test/queryClient.test.ts

```diff
@@ -1,36 +1,36 @@
 // Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { GraphqlQueryClient } from '../packages/network-clients';
-import { NETWORK_CONFIGS } from '../packages/network-clients/src';
+import { GetAllDelegations, GraphqlQueryClient } from '@subql/network-clients/src';
```

Comment:
Can we get these from `@subql/network-query`

Reply:
Set alias in monorepo tsconfig to solve this.